server/grpc/server_test.go

```diff
@@ -16,6 +16,8 @@ import (
 	"google.golang.org/grpc/metadata"
 	rpb "google.golang.org/grpc/reflection/grpc_reflection_v1alpha"
 
+	reflectionv1 "github.com/cosmos/cosmos-sdk/client/grpc/reflection"
```

Comment:
I believe reflection goes belongs indeed more into `server`, not sure why it was initially in client. Should we move that folder to `server`?

Reply:
I can move it into server